src/z3c/autoinclude/utils.txt

```diff
@@ -27,7 +27,7 @@ by having been extended by nested packages) it should find a package::
 
     >>> import enolp
     >>> distributionForPackage(enolp)
-    enolp.ppa.foo 0.1 (...enolp.ppa.foo-0.1...egg)
+    enolp.ppa.bar 0.1 (...enolp.ppa.bar-0.1...egg)
```

Comment:
Good point. It seems like ignoring the order that entries are on sys.path could have consequences, but I'll trust the comment that says they're equivalent.

Done.